File: app/src/main/java/org/lemonadestand/btb/features/login/models/WidgetSettings.java
package org.lemonadestand.btb.features.login.models;

import com.google.gson.annotations.SerializedName;

public class WidgetSettings {
    @SerializedName("button")
    Button button;

    @SerializedName("values")
    Values values;

    @SerializedName("form")
    Form form;


    public void setButton(Button button) {
        this.button = button;
    }
    public Button getButton() {
        return button;
    }

    public void setValues(Values values) {
        this.values = values;
    }
    public Values getValues() {
        return values;
    }

    public void setForm(Form form) {
        this.form = form;
    }
    public Form getForm() {
        return form;
    }


}

File: app/src/main/java/org/lemonadestand/btb/models/WidgetSettings.kt
package org.lemonadestand.btb.models

import com.google.gson.annotations.SerializedName

class WidgetSettings {
	@SerializedName("button")
	var button: Button? = null

	@SerializedName("values")
	var values: Values? = null

	@SerializedName("form")
	var form: Form? = null
}
